README.md

```diff
@@ -210,6 +210,8 @@ To create a PR for this project and start contributing follow this step-by-step
 
     ```sh
     $ cd visual-testing
+    $ corepack enable
+    $ corepack use pnpm@8.x
```

Comment:
If you run this locally it might set `"packageManager"` in *package.json*. We should commit that too. Unfortunately it isn't working on my ARM64 Linux box tonight. :facepalm: